available_images.md

```diff
@@ -82,17 +82,19 @@ You can pin your version by adding the version tag to your URL as follows:
 
      763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-cpu-py37-ubuntu18.04-v1.0
 
+
 | Framework         |Job Type 			     |Horovod Options 			       |CPU/GPU 	|Python Version Options     |  			 Example URL 			                                                                 | Latest Version|
 |-------------------|------------------------|---------------------------------|------------|---------------------------|----------------------------------------------------------------------------------------------------| --------------|
-|TensorFlow 2.2.0   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu102-ubuntu18.04   |v2.0           |
-|TensorFlow 2.2.0   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04   |v1.1           |
-|TensorFlow 2.2.0   |training                |Yes 	                           |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-cpu-py37-ubuntu18.04         |v1.1           |
-|TensorFlow 2.2.0   |inference               |No 	                           |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.2.0-gpu-py37-cu102-ubuntu18.04  |v1.0           |
-|TensorFlow 2.2.0   |inference               |No 	                           |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.2.0-cpu-py37-ubuntu18.04        |v1.0           |
+|[TensorFlow 2.2.0](https://github.com/aws/deep-learning-containers/releases/tag/v2.0-tf-2.2.0-tr-py37-gpu)   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu102-ubuntu18.04   |v2.0           |
+|[TensorFlow 2.2.0](https://github.com/aws/deep-learning-containers/releases/tag/v1.1-tf-2.2.0-tr-py37)   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04   |v1.1           |
```

Comment:
We have also released v1.2 images as well now. Should we include in new PR.

Reply:
Sounds good to me.

Reply:
@IvyBazan While linking to github releases is a good idea, the links may not always point to the latest images. We should probably hold this PR/close it till we have a better idea to link to the latest images. Till then, the image URIs that we already have should suffice.

Reply:
@IvyBazan Closing this PR for above reasons. Thanks for creating the PR!